sdks/typescript/src/apache_beam/worker/operators.ts

```diff
@@ -696,15 +697,15 @@ class GenericParDoOperator implements IOperator {
     }
   }
 
-  process(wvalue: WindowedValue<unknown>) {
+  async process(wvalue: WindowedValue<unknown>) {
```

Comment:
So even if this doesn't return a promise, it always creates one (and does the other async overhead), right?